lib/src/main/java/com/orange/ods/compose/component/textfield/password/OdsPasswordTextField.kt

```diff
@@ -50,9 +50,9 @@ import com.orange.ods.compose.component.utilities.UiModePreviews
  * field can not be modified, however, a user can focus it and copy text from it. Read-only text
  * fields are usually used to display pre-filled forms that user can not edit.
  * @param label Label to be displayed inside or outside the text field. The default text style used
- * is [Typography.caption] when the text field is in focus and [Typography.subtitle1] when the text field is not in focus.
+ * is [Typography.bodyS] when the text field is in focus and [Typography.titleM] when the text field is not in focus.
```

Comment:
The label is in `bodyS` when it has the focus (displayed above the text field), otherwise it is `titleM` in the text field if there is no text inside/